scheduler/ipp.c

```diff
@@ -5356,6 +5356,24 @@ create_local_bg_thread(
     }
   }
 
+  if (ippFindAttribute(response, "media-col-database", IPP_TAG_ZERO) == NULL
```

Comment:
The “media-col-database” and “media-size-supported” attributes are of type `IPP_TAG_COLLECTION` so you should use that instead of `IPP_TAG_ZERO`.

Reply:
@michaelrsweet so then we have to fix the line before that, where we use the tag zero as well. On it.